de-de/c-de.html.markdown

```diff
@@ -0,0 +1,880 @@
+> sollte die Man-Page des C-Compilers aufgerufen werden (z.B. `man 1 gcc`).
+> Alternativ kann auch online nach den unterschiedlichen Optionen gesucht werden.
+
+```c
+// einzeilige Kommentare starten mit // - nur in C99 und später vorhanden.
+
+/*
+mehrzeilige Kommentare sehen so aus. Diese funktionieren auch in C89
+*/
+
+/* 
+mehrzeilige Kommentare können nicht verschachtelt werden /* Sei Vorsichtig! */ // Kommentar endet auf dieser Linie ...
+*/ // ... nicht bei dieser!
+
+// Konstanten: #define <keyword>
+// Konstanten werden laut der Konvention immer in GROSSBUCHSTABEN geschrieben
+#define TAGE_IM_JAHR 365
+
+// Konstanten können auch als Aufzählungskonstanten (Enums) definiert werden.
+// Alle Anweisungen müssen mit einem Semikolon beendet werden.
+enum tage {SO=1, MO, DI, MI, DO, FR, SA};
+// MO wird automatisch zu 2, DI zu 3 etc.
+
+// Importiere Header-Dateien mit #include
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+// Dateien, welche zwischen <spitzen Klammern> stehen, sind Header-Dateien aus
+// der C-Standard-Bibliothek.
+// Für deine eigenen Header müssen Anführungszeichen verwendet werden, z.B.:
+// #include "mein_header.h"
+
+// Funktionssignaturen werden entweder vorher in einer .h-Datei deklariert oder
+// am Anfang der .c-Datei.
+void funktion_1();
+int funktion_2(void);
+
+// Es muss ein Funktionsprototyp deklariert werden vor der `main()` Funktion,
+// wenn die Funktion nach der `main()` Funktion gebraucht wird.
+int addiere_zwei_integer(int x1, int x2); // Funktionsprototyp
+// Auch wenn der Ausdrck `int addiere_zwei_integer(int, int)` auch valid wäre, 
+// ist es empfohlen, dass man die Namen der Argumente hinschreibt für eine
+// einfachere Analyse.
+
+// Der Einstiegspunkt deines Programms ist eine Funktion mit dem Namen main und 
+// einem Integer als Rückgabewert.
+int main(void){
+    // dein Programm
+}
+
+// Die Kommandozeilenargumente, welche gebraucht werden, damit dein Programm 
+// läuft, werden als Argumente der `main`-Funktion mitgegeben.
+// argc steht für die Anzahl von Argumenten. - Der Programmname ist das erste
+// Argument.
+// argv ist ein Array von Zeichenarrays, welche die Argumente beinhaltet.
+// argv[0] = Name des Programms
+// argv[1] = erstes Argument usw.
+int main (int argc, char** argv){
+    // Ausgabe mit Hilfe von printf (print formatted)
+    // %d ist ein Integer.
+    // \n steht für eine neue Zeile
+    printf("%d\n",0); // => Gibt 0 aus.
+
+    ////////////////////////////////////////////////
+    // Typen
+    ////////////////////////////////////////////////
+
+    // Alle Variablen müssen am Anfang des jetzigen Blocks deklariert werden.
+    // Wir deklarieren die Variablen dynamisch im Code um die Lesbarkeit im
+    // Tutorial zu verbessern.
+    // C99-Konforme Compiler erlauben die Variablendeklaration an dem Punkt, an
+    // welchem die Variable verwendet wird.
+
+    // integer sind normalerweise 4 Bytes gross
```

Comment:
instead of "gross" it should be "groß" for all occurences

Reply:
Ah, that's true. Didn't think about that because in Switzerland we don't know "ß" in words. 

Nevertheless I changed it, thanks!

Reply:
I went through the content of the tutorial and searched for every word which contains `ss`and changed it to `ß`. 

To fix this I used the website duden.de.